### ISSUE TITLE
Strange histograms

### ISSUE THREAD

**Issue by jojonki** [lanpa/tensorboard-pytorch#56]
Hi, this is really nice project. I started to use this library because I wanted to see histograms of my trainable parameters.

Although I run `demo.py`, the histograms looks strange. Is it expected behaviors?

- tensorboardX: 0.8
- python 3.6.1 (Anaconda)
- tensorflow: 1.4.0
- tensorflow-tensorboard: 0.4.0rc3
- PyTorch: 0.3.0.post4

```
$ python demo.py
$ tensorboard --logdir runs
```
<img width="1391" alt="screen shot 2017-12-08 at 3 53 50 pm" src="https://user-images.githubusercontent.com/166852/33784776-0d3f072a-dc30-11e7-8519-1265ab217b51.png">

**Comment by lanpa** [comment]
Please try https://github.com/lanpa/tensorboard-pytorch/issues/42 
I also noticed that you have a value 3*10^18, which is unusually big. Did the gradient explode?

**Comment by jojonki** [comment]
Do you mean to set `bins` like `bins=np.arange(-0.003, 0.003, 0.0001)`? Ok, I'll try it.
But I am afraid your official code doesn't work in some environment. Other libraries in my environment may be quite new.

**Comment by lanpa** [comment]
No, I think your value is too concentrated, therefore default binning method can not plot well. You can check this by printing the max and min of your data in console. And sorry I can't quite understand your second sentence. Is there error message?

**Comment by jyzhang-bjtu** [comment]
I got the same strange histogram too...

**Comment by jyzhang-bjtu** [comment]
Got it. The function add_histogram should use bins='auto' as default other than bins='tensorflow'.
The para bins='tensorflow' always cause such strange histogram

**Comment by lanpa** [comment]
@jyzhang-bjtu 'auto' might freeze the code. see #1 and use with care:)
